### ISSUE TITLE
Issue due to react-scripts version?

### ISSUE THREAD

**Issue by virenb** [fullstack-hy2020/unicafe-redux#2]
Hello,

First off, thank you for the wonderful course and curriculum. I'm currently on part 6, starting the exercises. For exercise 6.1, we are to clone this repository. I'm running into the below error when running `npm start` -

`TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined`

According to this StackOverflow question, I believe it is due to the version of `react-scripts`
[https://stackoverflow.com/questions/60234640/typeerror-err-invalid-arg-type-the-path-argument-must-be-of-type-string-re](https://stackoverflow.com/questions/60234640/typeerror-err-invalid-arg-type-the-path-argument-must-be-of-type-string-re)

It seems changing the dependency version of `react-scripts` will fix this problem. Not sure if others were having an issue, but would be happy to open a PR to fix this.

**Comment by Kaltsoon** [comment]
Hi!

All I can say is the classic: "works on my computer". However, using the latest `react-scripts` version (3.4.4) is generally a good idea. If you can open a pull request, we would gladly accept it.

**Comment by virenb** [comment]
Very true. I guess it worked for most people as there was no other issue filed regarding this. Any way, thank you for your message. I have created a pull request, updating the version.

I noticed the testing-library packages could have been updated as well, I wasn't sure if there were any 'breaking' changes, so I just updated react-scripts for now. I can look into updating them and test it out locally if you'd like.
